release.sh

```diff
@@ -2,32 +2,61 @@
 
 set -e
 
-last_tag=$(git tag --sort=-creatordate | head -n 1)
+last_tag=$(git tag --sort=-creatordate| head -n 1)
 new_tag=$(semver bump patch "$last_tag")
 timestamp=$(date -u +%Y%m%d%H%M%S)
 ref=$(echo "$BRANCH" | sed -e "s/[^a-zA-Z0-9]//g")
 new_version=$(if [ "$ref" = "master" ]; then echo "${new_tag}"; else echo "${new_tag}-${ref}.$timestamp"; fi)
 
-echo "Bump version: ${last_tag} -> ${new_version}"
-git tag "${new_version}"
-
-# Create image tag
-image_tag="${IMAGE_NAME}:${new_version}"
-docker tag "${IMAGE_NAME}" "${image_tag}"
-
-echo "Tagged docker image $image_tag"
-
-# If the branch is master, publish it.
-if [ "$ref" = "master" ]; then
+validateFileChanges() {
```

Comment:
Also, can you rename all the functions to use `snake_case` naming convention in lowercase instead of camelCase? This makes is consistent with `create_tag` function you've created too.